ccd-definition/AuthorisationCaseState/AuthorisationCaseState-nonprod.json

```diff
@@ -18,5 +18,24 @@
         "CRUD": "R"
       }
     ]
-  }
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseStateID": "JUDICIAL_REFERRAL",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "tribunal-caseworker",
+          "hearing-center-admin"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-civil-admin",
+          "[APPLICANTSOLICITORONE]"
+        ],
+        "CRUD": "R"
+      }
+    ]
```

Comment:
There should be a closing curly braces here. This json is not a valid json, that is why the build is failing at high level data setup stage.